vchernyshoff/homework_5/Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: test
+test:
+	@pytest tests
+
+.PHONY: install_requirements
+install_requirements:
+	@pip install --no-cache-dir --upgrade pip && pip install --no-cache-dir -r requirements.txt
+
+#install_requirements_dev
+#lint
+#black
+#pytest_coverage
```

Comment:
could you implement at least 2 commands from that list?

Reply:
@DmitryPoliuha added